quickstarts/gradle/micronaut-customized-image/build.gradle

```diff
@@ -0,0 +1,67 @@
+plugins {
+    id("com.github.johnrengelman.shadow") version "7.0.0"
+    id("io.micronaut.application") version "1.5.4"
+    id("org.eclipse.jkube.kubernetes") version "1.5.0-SNAPSHOT"
+}
+
+version = "1.5.0-SNAPSHOT"
+group = "org.eclipse.jkube.quickstart.gradle.micronaut.custom"
+
+repositories {
+    mavenCentral()
+    mavenLocal()
+}
+
+micronaut {
+    runtime("netty")
+    processing {
+        incremental(true)
+        annotations("org.eclipse.jkube.quickstart.gradle.micronaut.custom.*")
+    }
+}
+
+dependencies {
+    implementation("io.micronaut:micronaut-http-client")
+    implementation("io.micronaut:micronaut-management")
+    implementation("io.micronaut:micronaut-runtime")
+    implementation("javax.annotation:javax.annotation-api")
+    runtimeOnly("ch.qos.logback:logback-classic")
+    implementation("io.micronaut:micronaut-validation")
+}
+
+application {
+    mainClass.set("org.eclipse.jkube.quickstart.gradle.micronaut.custom.Application")
+}
+
+java {
+    sourceCompatibility = JavaVersion.toVersion("11")
+    targetCompatibility = JavaVersion.toVersion("11")
+}
+
+def imageName = project.getGroup() + "/" + project.getName() + ":" + project.getVersion()
+def javaAppFile = project.getName() + "-" + project.getVersion() + "-all.jar"
+def tagsList = ["latest", project.getVersion()]
+def sourceFile = "build/libs/" + javaAppFile
```

Comment:
We really want to avoid this.
We should be able to achieve this with template strings, let me check...

Reply:
I just added an extra change in #830 that would take care of this and allow defining things such as:
```
        image {
            name = "${project.group}/${project.name}:${project.version}"
```
Once merged and rebased you should be able to change your example and inline everything